### PR TITLE
Enable com.ibm.ws.microprofile.rest.client_fat to run on MP5.0 and 60 (EE9/EE10)

### PR DESCRIPTION
**Pull request by jim-krueger**

### CODE REVIEW

**Review by benjamin-confino — DISMISSED** [review]

**Comment by jim-krueger** [comment]
#libby #build

**Comment by jim-krueger** [comment]
#build

File: dev/com.ibm.ws.microprofile.rest.client_fat/test-applications/cdiPropsAndProvidersApp/src/mpRestClient11/cdiPropsAndProviders/CdiPropsAndProvidersTestServlet.java [hunk 1]
```diff
@@ -30,6 +30,8 @@
 import org.junit.Test;
 
 import componenttest.app.FATServlet;
+import componenttest.rules.repeater.JakartaEE10Action;
+import componenttest.rules.repeater.JakartaEE9Action;
```
Comment (jhanders34) [inline dev/com.ibm.ws.microprofile.rest.client_fat/test-applications/cdiPropsAndProvidersApp/src/mpRestClient11/cdiPropsAndProviders/CdiPropsAndProvidersTestServlet.java]:
```suggestion
```

**Review by jhanders34 — DISMISSED** [review]

File: dev/com.ibm.ws.microprofile.rest.client_fat/test-applications/cdiPropsAndProvidersApp/src/mpRestClient11/cdiPropsAndProviders/CdiPropsAndProvidersTestServlet.java (hunk 1 above)
Comment (jim-krueger) [reply dev/com.ibm.ws.microprofile.rest.client_fat/test-applications/cdiPropsAndProvidersApp/src/mpRestClient11/cdiPropsAndProviders/CdiPropsAndProvidersTestServlet.java]: thanks

**Comment by jim-krueger** [comment]
#build

**Review by WhiteCat22 — APPROVED** [review]

**Review by jhanders34 — APPROVED** [review]

**Comment by jim-krueger** [comment]
#libby

**Comment by jim-krueger** [comment]
#run-libby-bot

**Comment by jim-krueger** [comment]
test failure in personal build is unrelated.   Merging

**Pull request merged**